src/Pages/Main/Main.js

```diff
@@ -0,0 +1,346 @@
+import React, { useState, useRef } from 'react';
+import { Link } from 'react-router-dom';
+import Modal from './Modal/Modal';
+import PassengerCount from './Modal/PassengerCount';
+import Calendar from './Modal/Calendar';
+import PickCity from './Modal/PickCity';
+import moment from 'moment';
+import 'moment/locale/ko';
+import styled from 'styled-components/macro';
+
+function Main() {
+  const [countOn, setCountOn] = useState(false);
+  const [calendarOn, setCalendarOn] = useState(false);
+  const [cityOn, setCityOn] = useState(false);
+  const [amount, setAmount] = useState(0);
+  const [depDate, setDepDate] = useState(moment().format('YYYY.MM.DD'));
+  const [arrDate, setArrDate] = useState(moment().format('YYYY.MM.DD'));
+  const [depLocation, setDepLocation] = useState(['ICN', '서울/인천']);
+  const [arrLocation, setArrLocation] = useState(['To', '도착지']);
```

Comment:
```suggestion
  const [departure, setDeparture] = useState({
    date: moment().format('YYYY.MM.DD'),
    location: ['ICN', '서울/인천']
  })
```
이런 느낌은 어떠세요?? state 중 묶어 사용할 수 있는건 묶어주는게 좋을듯 해요

Reply:
화이팅...